log4j-nosql/src/main/java/org/apache/logging/log4j/nosql/appender/mongodb/MongoDbProvider.java

```diff
@@ -97,6 +103,8 @@ public String toString() {
     @PluginFactory
     public static MongoDbProvider createNoSqlProvider(
             @PluginAttribute("collectionName") final String collectionName,
+            @PluginAttribute(value = "capped", defaultBoolean = false) final Boolean isCapped,
```

Comment:
Hi @jvz, I changed the `Boolean` member to be a primitive type. But I don't know what kind of Builder you mean for the attributes? Does this builder exist for the MongoDbProvider? Cheers Matt